public/src/css/style.css

```diff
@@ -2042,6 +2042,11 @@ ol.search_list > li::before {
     color: #555;
 }
 
+span.also a{
```

Comment:
Thanks @piuccio Further down in the collection html there are other links which picked up the red styling, so tried to add the specificity to lear it up.